R/tm_a_regression.R

```diff
@@ -33,13 +33,12 @@
 #' @examples
 #' # Regression graphs from selected response variable (BMRKR1) and
 #' # selected regressors (AGE)
-#' library(scda)
 #'
-#' ADSL <- synthetic_cdisc_data("latest")$adsl
+#' ADSL <- rADSL
 #'
 #' app <- init(
 #'   data = cdisc_data(
-#'     cdisc_dataset("ADSL", ADSL, code = "ADSL <- synthetic_cdisc_data(\"latest\")$adsl"),
+#'     cdisc_dataset("ADSL", ADSL, code = "ADSL <- rADSL"),
```

Comment:
```suggestion
#'     teal.data::cdisc_dataset("ADSL", ADSL, code = "ADSL <- rADSL"),
```
Can you append package name in front of all functions in examples? So it's clear this functions comes from `teal.data` and not `scda`. It is easier for debugging. Can you apply this comment to all examples in this package?